### ISSUE TITLE
Kotlin project, could not find mainactivity.java file error

### ISSUE THREAD

**Issue by baryon** [quasarframework/quasar#7557]
**Describe the bug**

build capacitor for android got an error.

```
App · Updated capacitor.config.json
App Error: could not find mainactivity.java file and therefore cannot enable devserver: https support
```
because our android project wrote by Kotlin, not Java. it dont include mainactivity.java

BUT, it was running Quasar 1.9.4. not working 2.0.6

iOS version is working.

**Expected behavior**
NO any error

**Platform (please complete the following information):**
OS: Mac OS 10.15.6
Node: ESM
NPM: 6.14.6
Yarn:
Browsers:
iOS:
Android: android studio 4.0.1
Electron:

**Additional context**
iOS version is working.

When I change the code https://github.com/quasarframework/quasar/blob/dev/app/lib/capacitor/capacitor-config.js Line. 231

from ```     let mainActivityPath = fglob.sync(`**/MainActivity.java`, { cwd: capacitorSrcPath, absolute: true }) ```
to ```     let mainActivityPath = fglob.sync(`**/MainActivity.*`, { cwd: capacitorSrcPath, absolute: true }) ```

it wroks. I dont know what is right

**Comment by rstoenescu** [comment]
Fix will be available in "@quasar/app" v2.0.7

**Comment by rstoenescu** [comment]
I'm really interested in how you created your project folder though (especially the capacitor part), if you'd be so kind @baryon . Thanks in advance.
